templates/patterns/copyright_overlay/pattern-copyright-overlay.html.twig

```diff
@@ -0,0 +1,57 @@
+{#
+/**
+ * @file
+ * Copyright overlay pattern.
+ */
+#}
+{{ attach_library('oe_bootstrap_theme/copy_clipboard') }}
+
+{% set _trigger_label = trigger_label|default('View more'|t) %}
+
+{% set _classes = ['copyright-overlay', 'px-2', 'text-muted'] %}
+
+{% set _footer %}
+  {% include '@oe-bcl/button' with {
+    label: 'Copy Copyright'|t,
+    attributes:  create_attribute()
+      .setAttribute('data-copy-target', 'copyright-text'),
+  } only %}
+{% endset %}
+
+{% set _header %}
+  <h5 class="mb-0">{{ 'Copyright'|t }}</h5>
+  {% include '@oe-bcl/button' with {
+    clean_class: true,
+    attributes:  create_attribute()
+      .addClass(['btn-close'])
+      .setAttribute('data-bs-dismiss', 'modal')
+      .setAttribute('aria-label', 'Close'),
+  } only %}
+{% endset %}
+
+{% set _body %}
+  <div class="copyright-text">
+    {{ text }}
+  </div>
+{% endset %}
+
+<div{{ attributes.addClass(_classes) }}>
+  <span class="preview">
+    {{ text }}
+  </span>
+  {{ pattern('link', {
+    label: _trigger_label,
+    path: '#',
+    attributes: create_attribute()
+      .setAttribute('data-bs-toggle', 'modal')
+      .setAttribute('data-bs-target', '#' ~ modal_id),
+  }) }}
+</div>
+{% include '@oe-bcl/modal' with {
+  'id': modal_id,
+  'header': _header,
+  'show_close_button': true,
```

Comment:
This is a field of the modal pattern, not really used like this.
I am a bit conflicted, should we not use the pattern? in case there are fixes or improvements it would resolve also here, feel this is half the reason to use patterns.

Reply:
i am using now the pattern.